pages/1.10/release-notes/1.10.4/index.md

```diff
@@ -15,7 +15,7 @@ These are the release notes for DC/OS 1.10.4.
 - CORE-1375 - Docker executor does not hang due to lost messages.
 - DOCS-2169	- Updated [ports list](https://docs.mesosphere.com/1.10/installing/ent/ports/) for DC/OS. [enterprise type="inline" size="small"/]
 - DCOS-18777 - DC/OS CA certificate bundle now gets propagated to public slaves.
-- DCOS-19327 - Diagnostics bundles no longer contain sensitive cluster configuration values.
+- DCOS-19327 - Diagnostics bundles no longer contain sensitive cluster configuration values related to Cloudformation templates.
```

Comment:
Note to self: in the future, release notes should contain a dedicated section about security-relevant changes.